sdk/timeseriesinsights/Azure.IoT.TimeSeriesInsights/src/Customized/Models/GetEvents.Serialization.cs

```diff
@@ -36,7 +36,7 @@ internal partial class GetEvents : IUtf8JsonSerializable
             {
                 writer.WritePropertyName("projectedProperties");
                 writer.WriteStartArray();
-                foreach (EventProperty item in ProjectedProperties)
+                foreach (TimeSeriesInsightsEventProperty item in ProjectedProperties)
```

Comment:
Now that I think about this, TimeSeriesInsightsEventProperty seems too long when we are not making changes to ProjectedProperties!